avocado/utils/distro.py

```diff
@@ -260,6 +263,17 @@ def get_distro(self):
 
         return distro
 
+    def log_version_detection_fail(self, version, distro):
+        """
+        Log an error when the version detection fails.
+
+        :param version: Detected version string (eg. '28')
+        :param distro: LinuxDistro object
+        """
+        self.log.error("Version detection failed for distro '%s', "
+                       "version string '%s'", distro.name, version)
+        self.log.error('Report this bug visiting %s', project.NEW_ISSUE_LINK)
```

Comment:
Honestly I don't think such thing is necessary as users should notify `UNKNOWN` version and yet another logged message shouldn't make much difference. But I'm not strictly against.